.github/workflows/jenkins_prepare.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       run:
         working-directory: ./jenkins
     steps:
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+      - name: Set up Cortex-M
+        uses: docker/setup-cortexm-action@v1
```

Comment:
This doesn't seem to exist (e.g. this is the original https://github.com/docker/setup-qemu-action), do we need to change this if the underlying qemu is still the same?